recipes.js

```diff
@@ -1,10 +1,110 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
-
-mongoose.connect('mongodb://localhost/recipeApp')
-  .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
-  });
+const mongoose = require("mongoose");
+const express = require("express");
+const app = express();
+app.use(require('easy-livereload')());
+
+//const Schema = mongoose.Schema;
+const chalk = require("chalk");
+
+const data = require("./data.js");
+const Recipe = require("./models/recipe-model.js");
+
+mongoose
+	.connect("mongodb://localhost/recipeApp")
+	.then(() => {
+		console.log(chalk.green("Connected to Mongo! ❤️"));
+	})
+	.catch(err => {
+		console.log(chalk.red.bold("Error connecting to Mongo....... 💩", err));
+	});
+
+// set server
+app.listen(3000, () => {
+	console.log("expressive llamas in pyjamas baby");
+});
```

Comment:
love your console log! 
The app.listen is not mandatory because the exercise was just manipulating mongoose. But it's no big deal though. :)

Reply:
aha I love [easter eggs](https://dev.to/philnash/developer-whimsy-llamas-in-pajamas-232i) !